CHANGELOG.rst

```diff
@@ -0,0 +1,9 @@
+0.3.3 — 2024-05-22
+------------------
+
+Fixed
+.....
+
+- `PR 50`_ The 'ValueError("The given username must be set")' when using Google OAuth 2.0 to login
```

Comment:
Changelog should refer to an issue that's been closed.Could you file a complete issue of the traceback of the error you are seeing with full steps to reproduce it?